DiscordBot/moderator.py

```diff
@@ -0,0 +1,260 @@
+                reply = self.create_options_list("This is the number of times the user has submitted fake reports in the past. How would you like to proceed?",
+                                                 self.FALSE_REPORT_DECISION_OPTIONS)
+                return [reply]
+            else:
+                self.state = ModeratorState.ACTION_COMPLETE
+                reply = "Report escalated to next moderator team."
+                replies += [reply]
+
+        if self.state == ModeratorState.AWAITING_FALSE_REPORT_DECISION:
+            i = self.get_index(message, self.FALSE_REPORT_DECISION_OPTIONS)
+            if i == -1:
+                return ["Please enter a number corresponding to the given options."]
+            
+            if i == 0:
+                self.state = ModeratorState.ACTION_COMPLETE
+                reply = "User received a warning message. We are closing the report."
+                replies += [reply]
+
+            if i == 1:
+                self.state = ModeratorState.ACTION_COMPLETE
+                reply = "We increased the number of false reports associated to this user. We are closing the report."
+                replies += [reply]
+                # TODO increase the number of times the user has submitted false reports
+        
+        if self.state == ModeratorState.AWAITING_LEGIT_REPORT_DECISION:
+            i = self.get_index(message, self.LEGIT_REPORT_DECISION_OPTIONS)
+            if i == -1:
+                return ["Please enter a number corresponding to the given options."]
+            
+            if i == 0:
+                await self.handle_escalate(report)
+                await self.handle_ban(report)
+                self.state = ModeratorState.ACTION_COMPLETE
+
+            if i == 1:
+                self.state = ModeratorState.AWAITING_REPORT_FREQUENCY_DECISION
+                reply = self.create_options_list("This is the number of times the user has been reported for potential CSAM. How would you like to proceed?",
+                                                 self.REPORT_FREQUENCY_DECISION_OPTIONS)
+                return [reply]
+                
+        if self.state == ModeratorState.AWAITING_REPORT_FREQUENCY_DECISION:
+            i = self.get_index(message, self.REPORT_FREQUENCY_DECISION_OPTIONS)
+            if i == -1:
+                return ["Please enter a number corresponding to the given options."]
+            
+            if i == 0:
+                await self.handle_escalate(report)
+                await self.handle_ban(report)
+                self.state = ModeratorState.ACTION_COMPLETE
+
+            if i == 1:
+                self.state = ModeratorState.AWAITING_HIDE_CHILDREN_DECISION
+                # TODO increase number of times user has been reported
+                reply = self.create_options_list("We increased the number of false reports associated to this user. Would you like to hide child profiles from this user?",
```

Comment:
We need to change the text to be: 

`We increased the number of times this user has been reported for CSAM. Would you like to hide child profiles from this user?`

Reply:
Changed.